### PR TITLE
fix up input file syntax

### PR DESCRIPTION
**Pull request by rwcarlsen**
necessary for upcoming parser improvements in idaholab/moose#9617

### CODE REVIEW

**Comment by amjokisaari** [comment]
@dschwen, please advise if this should be merged given the discussion on [idaholab/moose#9617](https://github.com/idaholab/moose/pull/9617)?

File: include/base/HyraxRevision.h [hunk 1]
```diff
@@ -3,6 +3,7 @@
 #ifndef HYRAX_REVISION_H
 #define HYRAX_REVISION_H
 
-#define HYRAX_REVISION "git commit d6ecb30 on 2017-05-11"
+#define HYRAX_REVISION "git commit 85cd787 on 2017-06-28"
+#define HYRAX_VERSION "85cd787"
```
Comment (dschwen) [inline include/base/HyraxRevision.h:7]: Wait, why is this file checked in in the first place?

**Comment by dschwen** [comment]
The changes all look good. I'd remove the revision file from the repo though (it is autogenerated). But that could be another PR.

**Review by permcody — APPROVED** [review]

**Comment by permcody** [comment]
Ugh, that file should be removed but I'm more interested in getting the syntax fixed.

**Pull request merged**